src/odemis/gui/win/acquisition.py

```diff
@@ -612,6 +613,409 @@ def on_acquisition_done(self, future):
+                                                        static=True,
+                                                        ignore_view=True)
+        # The streams currently displayed are the one visible
+        self.add_streams()
+
+        # The list of streams ready for acquisition (just used as a cache)
+        self._acq_streams = {}
+
+        # FIXME: pass the fold_panels
+
+        # Compute the preset values for each preset
+        self._orig_entries = get_global_settings_entries(self._settings_controller)
+        self._orig_settings = preset_as_is(self._orig_entries)
+        for sc in self.streambar_controller.stream_controllers:
+            self._orig_entries += get_local_settings_entries(sc)
+
+        self.start_listening_to_va()
+
+        # make sure the view displays the same thing as the one we are
+        # duplicating
+        self._view.view_pos.value = orig_view.view_pos.value
+        self._view.mpp.value = orig_view.mpp.value
+        self._view.merge_ratio.value = orig_view.merge_ratio.value
+
+        # attach the view to the viewport
+        self.pnl_view_acq.canvas.fit_view_to_next_image = False
+        self.pnl_view_acq.setView(self._view, self._tab_data_model)
+
+        # The TOOL_ROA is not present because we don't allow the user to change
+        # the ROA), so we need to explicitly request the canvas to show the ROA.
+        if hasattr(self._tab_data_model, "roa") and self._tab_data_model.roa is not None:
+            cnvs = self.pnl_view_acq.canvas
+            self.roa_overlay = RepetitionSelectOverlay(cnvs, self._tab_data_model.roa,
+                                                             self._tab_data_model.fovComp)
+            cnvs.add_world_overlay(self.roa_overlay)
+
+        self.Bind(wx.EVT_CHAR_HOOK, self.on_key)
+
+        self.btn_cancel.Bind(wx.EVT_BUTTON, self.on_close)
+        self.btn_secom_acquire.Bind(wx.EVT_BUTTON, self.on_acquire)
+        self.Bind(wx.EVT_CLOSE, self.on_close)
+        # on_streams_changed is compatible because it doesn't use the args
+
+        # To update the estimated time when streams are removed/added
+        self._view.stream_tree.flat.subscribe(self.on_streams_changed)
+
+        # set parameters for tiled acq
+        # TODO: avoid hardcoding, use stage metadata
+        fm_fov = compute_camera_fov(self._main_data_model.ccd)
+        # Using "songbird-sim-ccd.h5" in simcam with tile max_res: (260, 348)
+        # self.area = MD_FAV_AREA from stage
+        self.area = (0, 0, fm_fov[0] * 2, fm_fov[1] * 2)  # left, top, right, bottom
+        self.overlap = 0.2
+
+        # Note: It should never be possible to reach here with no streams
+        streams = self.get_acq_streams()
+        v_streams = self._view.getStreams()  # visible streams
+        for s in streams:
+            # Add extra viewable streams to view. However, do not add incompatible streams.
+            if s not in v_streams and not isinstance(s, NON_SPATIAL_STREAMS):
```

Comment:
This whole code seems an overly complicated way to do nothing, as `.get_acq_streams()` essentially returns `view.getStreams()`. => Drop?